Topics/Tech_Stacks/Swift_Mapkit.md

```diff
@@ -0,0 +1,263 @@
+# Introduction to MapKit for SwiftUI
+
+SwiftUI is a very vast collection of features designed by Apple to make iOS development…swift-er. One of those handy features is it’s ability to make maps, specifically with MapKit. 
+
+**Requirements:** To use these MapKit features, you need to use Xcode 15 and later.
+
+## Table of contents
+### [What is MapKit?](#what-is-mapkit-1)
+### [Creating a map](#creating-a-map-1)
+### [Markers](#markers-1)
+### [Annotations](#annotations-1)
+### [Camera Position](#camera-position-1)
+### [Additional Resources](#additional-resources-1)
+
+## What is MapKit?
+MapKit is one of Apple’s API framework that allows iOS developers to build map-centered views and apps easily and efficiently. 
+
+Based on the [Apple documentation](https://developer.apple.com/documentation/mapkit/), MapKit can:
+
+- Embed maps directly into your app’s windows and views.
+- Add annotations and overlays to a map to call out points of interest.
+- Add LookAround capabilities to enable users to explore locations at street level.
+- Respond to user interactions with well known points of interest, geographical features, and boundaries.
+- Provide text completion to make it easy for users to search for a destination or point of interest.
+## Creating a Map
+Let’s start with a basic map.
+<p align="center">
+  <img width=1000 src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/113125436/15f46149-bc31-4b53-98d5-0e67b44abb26"/>
+</p>
+
+To create a basic map, you first need to import MapKit into your Swift file. Then simply call Map() in the view. 
+
+```swift
+import SwiftUI
+import MapKit
+
+struct MapContentView: View {
+    var body: some View {       
+            Map()
+    }
+}
+
+#Preview {
+    MapContentView()
+}
+```
+
+
+Congratulations you made a map! As you can tell this is just the bare bones. Now that you have a map, you have to customize it to fit your needs.
+
+### Customizing your map
+
+**Map Style**
```

Comment:
```suggestion
**Map Style**
```
Needs another inline after the title, it isn't displaying properly in the HTML render